pkg/apk/install.go

```diff
@@ -209,6 +220,18 @@ func (a *APK) installAPKFiles(gzipIn io.Reader, origin, replaces string) ([]tar.
 		default:
 			return nil, fmt.Errorf("unsupported file type %s %v", header.Name, header.Typeflag)
 		}
+
+		// xattrs
+		for k, v := range header.PAXRecords {
+			if !strings.HasPrefix(k, "SCHILY.xattr.") {
+				continue
+			}
+			attrName := strings.TrimPrefix(k, "SCHILY.xattr.")
```

Comment:
Should this be using the constant defined above?

Reply:
Yeah, it should. Separate PR coming